### PR TITLE
delete ns listener

### PR DESCRIPTION
**Pull request by YaoZengzeng**

### CODE REVIEW

File: pkg/hyper/network.go [hunk 1]
```diff
@@ -17,140 +17,70 @@ limitations under the License.
...
 
 	"github.com/containernetworking/cni/pkg/ns"
+	cnitypes "github.com/containernetworking/cni/pkg/types"
+	"github.com/containernetworking/cni/pkg/types/current"
 	"github.com/vishvananda/netlink"
 )
 
-type NetNsInfos struct {
-	Interfaces []*InterfaceInfo
+type NetworkInfo struct {
```
Comment (feiskyer) [inline pkg/hyper/network.go]: Add comments for public types

File: pkg/hyper/network.go [hunk 2]
```diff
@@ -17,140 +17,70 @@ limitations under the License.
...
+		return nil
 	}
 
-	for _, link := range links {
-		if link.Type() == "lo" {
-			// only omit "lo" here
+	// only handle bridge plugin now
+	for _, iface := range r.Interfaces {
+		// veth in host
+		if strings.Contains(iface.Name, "veth") {
```
Comment (feiskyer) [inline pkg/hyper/network.go]: Doesn't this check really required? I think `iface.Sandbox != ""` is enough for getting container interfaces

**Comment by YaoZengzeng** [comment]
Already fixed, ready for review @feiskyer

File: pkg/hyper/network.go [hunk 3]
```diff
@@ -17,140 +17,72 @@ limitations under the License.
...
 	if err != nil {
-		return infos
+		glog.Errorf("Convert CNI Result failed: %v", err)
+		return nil
 	}
 
-	for _, link := range links {
-		if link.Type() == "lo" {
-			// only omit "lo" here
+	// only handle bridge plugin now
```
Comment (feiskyer) [inline pkg/hyper/network.go]: Could you add a TODO for supporting other plugins?

File: pkg/hyper/ocicni/ocicni.go [hunk 4]
```diff
@@ -29,8 +29,6 @@ import (
 )
 
 type cniNetworkPlugin struct {
```
Comment (feiskyer) [inline pkg/hyper/ocicni/ocicni.go]: Add a TODO of upgrading cni plugin to its stable release when v0.5.0 is released

**Comment by YaoZengzeng** [comment]
Ready for review again @feiskyer

**Comment by feiskyer** [comment]
LGTM. Thanks.

**Pull request merged**